services/microcredit/src/notification.ts

```diff
@@ -228,4 +229,65 @@ export const _sendPushNotification = async (usersToNotify: number[], type: numbe
         true,
         { path: interfaces.app.appNotification.NotificationParamsPath.LOAN_APPROVED }
     );
+}
+
+export const reachingMaturity = async () => {
+    const borrowers = await database.models.subgraphMicroCreditBorrowers.findAll({
+        attributes: [],
+        where: literal(`
+            DATE(TO_TIMESTAMP(claimed + period) - INTERVAL '7 days') = DATE(CURRENT_TIMESTAMP) 
+         OR DATE(TO_TIMESTAMP(claimed + period) - INTERVAL '1 days') = DATE(CURRENT_TIMESTAMP)
+        `),
+        include: [{
+            attributes: ['language', 'email'],
+            model: database.models.appUser,
+            as: 'user',
+            required: true,
+            where: {
+                email: { [Op.not]: null }
+            }
+        }]
+    });
+    const languages = [...new Set(borrowers.map(el => el.user.language))];
+
+    const fetchNotificationsFromPrismic = async (language: string) => {
+        const borrowersToNotify = borrowers.filter(el => el.user.language === language);
+        const locale = utils.locales.find(({ shortCode }) => language === shortCode.toLowerCase())?.code;
+        const defaultLocale = utils.locales.find(({ isDefault }) => isDefault)?.code;
+
+        // get prismic document
+        const response = await utils.prismic.client.getAllByType('push_notifications_data', {
+            lang: locale || defaultLocale
+        });
+
+        const data = response[0].data;
+        const baseKey = ''
```

Comment:
I see baseKey is empty. Did you created the strings on prismic?

Reply:
I created the text field on prismic, but I left blank